### ISSUE TITLE
Full configuration not passed to view on initial render

### ISSUE THREAD

**Issue by edcrampin** [Anyline/anyline-ocr-react-native-module#29]
Hi,

I've been playing round with the config and playing round with values as specified in the documentation, however it seems that upon the initial call of the Anyline setupScanViewWithConfigJson, not all the config options follow through in to the view produced.

An example of this is flash alignment, stroke colours for the cut out (these were both referenced in #22 ), certain offsets, and any modifications to the visual feedback. However, some options within the config are in fact passed through, such as the done button related properties.

If I include a segment to change mode, upon selecting a different mode the full configuration as per my config.js file loads, indicating that it is not fully loaded on the initial render.

Thanks.

Here's my current config:
```
{
    options: {
        captureResolution: '1080',
        cutout: {
            style: 'rect',
            alignment: 'top',
            offset: {
                'x': 0,
                'y': 120,
            },
            strokeWidth: 1,
            cornerRadius: 4,
            strokeColor: 'ff0099',
            feedbackStrokeColor: 'ff0099',
            outerColor: '000000',
            outerAlpha: 0.5,
        },
        flash: {
            mode: 'manual',
            alignment: 'bottom_right'
        },
        doneButton: {
            title: 'Cancel',
            type: 'rect',
            cornerRadius: 0,
            textColor: "ffffff",
            textColorHighlighted: "cccccc",
            fontSize: 24,
            fontWeight: "100",
            positionXAlignment: "center",
            positionYAlignment: "bottom",
            offset: {
                "x": 0,
                "y": -200,
            }
        },
        beepOnResult: true,
        vibrateOnResult: true,
        blinkAnimationOnResult: true,
        cancelOnResult: true,
        reportingEnabled: false,
        visualFeedback: {
            style: "contour_rect",
            strokeColor: "ff0099",
            strokeWidth: 1,
            fillColor: "22FF0099",
            cornerRadius: 4,
            redrawTimeout: 200,
            animation: "TRAVERSE_MULTI",
            animationDuration: 75
        }
    },
    nativeBarcodeEnabled: true
}
```

**Comment by JonasLaux** [comment]
Hi,
sorry for the late response, i will look at this. On which platform you're getting this issues?

**Comment by JonasLaux** [comment]
I think i found the bug. I will fix this with our next release 3.20

**Comment by edcrampin** [comment]
Great, thanks Jonas.
